package.json

```diff
@@ -110,9 +110,7 @@
   },
   "browserslist": {
     "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
+      "extends @gravity-ui/browserslist-config"
```

Comment:
It seems there's no sense to split setting into production and development. Let's combine these parts into one.

Reply:
Done!